modules/oxide/src/less/theme/content/footnotes/footnotes.less

```diff
@@ -0,0 +1,32 @@
+//
+// Footnotes
+//
+
+sup.mce-footnote a {
+  color: black;
+  text-decoration: none;
+}
+
+div.mce-footnotes {
+  hr {
+    margin-left: 0;
+    margin-right: auto;
+    width: 25%;
+  }
+
+  li > a:first-child {
+    color: black;
+    text-decoration: none;
+  }
+}
+
+@media print {
+  div.mce-footnotes {
+    break-inside: avoid;
+    width: 100%;
+
+    li > a:first-child {
```

Comment:
Which element does this hide?

Reply:
The '^' symbol used to navigate back to the footnote in the content.

Reply:
I think, similar to @ephox-mogran's comment above, it should have a unique class. Please coordinate with @ltrouton on what the class name should be.